esphome/components/ltr303/ltr303.cpp

```diff
@@ -0,0 +1,322 @@
+#include "ltr303.h"
+#include "esphome/core/application.h"
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+
+namespace esphome {
+namespace ltr303 {
+
+static const char *const TAG = "ltr303";
+static const int ALS_GAIN[8] = {1, 2, 4, 8, 0, 0, 48, 96};                        // multiplier
```

Comment:
These could be `uint8_t` -- would save some memory...
```suggestion
static const uint8_t ALS_GAIN[8] = {1, 2, 4, 8, 0, 0, 48, 96};                        // multiplier
```

Reply:
thats my bad, copy paste from my test code. fixed